presence-detector.py

```diff
@@ -69,6 +69,13 @@ def __init__(self, config_file: str) -> None:
         self._watchers: List[UbusWatcher] = []
         self._killed = False
 
+    @staticmethod
+    def post(url: str, json: dict = {}, headers: dict = {}, timeout: int = None):
```

Comment:
please change to _post and rename the json param so the nasty globals() trick is unneeded. Alternative: import json as pyjson (or something) and use that.